### PR TITLE
updated versions & improved car-processing

### PR DESCRIPTION
**Pull request by ajhunyady**

### CODE REVIEW

**Comment by ajhunyady** [comment]
I mistakenly removed the `visualizer` but I don't know how to remove it from the pull request. Some help would be appreciated.

File: dataflows/helsinki-transit/connector/helsinki-mqtt.log [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[2m2024-12-18T14:21:25.503236Z[0m [32m INFO[0m [2mmqtt_source[0m[2m:[0m Using EnvSecretStore
```
Comment (morenol) [inline dataflows/helsinki-transit/connector/helsinki-mqtt.log]: Do we need this file?

File: dataflows/car-processing/README.MD [hunk 2]
```diff
@@ -15,49 +15,42 @@ The following diagram is a visual representation generated by `sdf`:
...
-  "timestamp": "2024-03-14T16:12:10.493Z"
-}
-```
-
-This connector sends events to `cars` topic.
-
-### License Plates Connector
-
-The second connector reads car licenses every hour to emulate a database read of license places from an official organization such as the Department of Motor Vehicles (DMV).
+The datasets is hosted by InfinyOn Synthetic data generator at `demo-data.infinyon.com`.
```
Comment (crajcan) [inline dataflows/car-processing/README.MD]:
```suggestion
The dataset is hosted by the InfinyOn Synthetic data generator at `demo-data.infinyon.com`.
```

File: dataflows/car-processing/README.MD [hunk 3]
```diff
@@ -15,49 +15,42 @@ The following diagram is a visual representation generated by `sdf`:
...
+
+* The `licenses` are a list of static license plate numbers, so we'll only read it once.
```
Comment (crajcan) [inline dataflows/car-processing/README.MD]:
```suggestion
* The `licenses` are a list of static license plate numbers, so we'll only read them once.
```

File: dataflows/car-processing/README.MD [hunk 4]
```diff
@@ -15,49 +15,42 @@ The following diagram is a visual representation generated by `sdf`:
...
+    "license": "7WPK493",
+    "fuel": "gasoline",
+    "location": "Saratoga",
+    "mph": 43,
+    "timestamp": "2024-03-14T16:12:10.493Z"
+  }
+  ```
+
 
 The connector sends events to `licenses` topic.
```
Comment (crajcan) [inline dataflows/car-processing/README.MD]:
```suggestion
The connector sends events to the `licenses` topic.
```

File: dataflows/helsinki-transit/connector/helsinki-mqtt.log (hunk 1 above)
Comment (ajhunyady) [reply dataflows/helsinki-transit/connector/helsinki-mqtt.log]: No we don't.  Removed

File: dataflows/car-processing/README.MD [hunk 5]
```diff
@@ -81,15 +74,25 @@ sdf run --ui
...
+```
+
+To see the licenses, run `fluvio consume licenses -Bd -O json`.
+
+
+### 3. Start the car connector:
 
 
-In a new terminal change direcotry to `./connectors`, download the connector binary, and start 2 connectors:
+In a new terminal change directory to `./connectors`, download the connector binary, and start 2 connectors:
```
Comment (crajcan) [inline dataflows/car-processing/README.MD]:
```suggestion
In a new terminal change directory to `./connectors`, download the connector binary, and start the connector:
```

It looks like the script only starts one now?

File: dataflows/car-processing/README.MD [hunk 6]
```diff
@@ -132,15 +135,51 @@ show state save-license-plates/licence-plates/state
...
+```
+
+There are two tables, but in this example we are interested in the `license_plates` table.
+Let's run a simple select:
+
+```sql
+select * from license_plates
+```
+
+Now we can look-up how many models does toyota have:
```
Comment (crajcan) [inline dataflows/car-processing/README.MD]:
```suggestion
Now we can look-up how many models toyota has:
```

File: dataflows/car-processing/README.MD [hunk 7]
```diff
@@ -317,7 +372,6 @@ Note that your connectors will continue to run in the background until you shut
 
 ```bash
 cdk deploy shutdown --name car-connector
-cdk deploy shutdown --name license-connector
```
Comment (crajcan) [inline dataflows/car-processing/README.MD:320]: perhaps the line above should now read "Note that your connector will continue to run in the background until you shut it down"

File: dataflows/car-processing/connectors/README.MD [hunk 8]
```diff
@@ -51,5 +34,4 @@ Stop the connectors:
 
 ```bash
 cdk deploy shutdown --name car-connector
-cdk deploy shutdown --name license-connector
```
Comment (crajcan) [inline dataflows/car-processing/connectors/README.MD:54]: Same here, the above instruction should be "Stop the connector:"

**Review by crajcan — CHANGES_REQUESTED** [review]

**Comment by ajhunyady** [comment]
@crajcan, done

**Review by crajcan — APPROVED** [review]
Nice.

**Pull request merged**